SimPEG/directives/directives.py

```diff
@@ -347,16 +348,18 @@ class BaseBetaEstimator(InversionDirective):
     ----------
     beta0_ratio : float
         Desired ratio between data misfit and model objective function at initial beta iteration.
-    seed : int, None
-        Seed used for random sampling.
+    seed : int, numpy.random.Generator or None, optional
```

Comment:
Technically I think we should support the same inputs as https://numpy.org/doc/stable/reference/random/generator.html#numpy.random.default_rng (and maybe point the user to this function for more details about the rng?)

Reply:
Makes sense. I'll update the docstrings.